gdal/port/cpl_http.cpp

```diff
@@ -758,27 +758,44 @@ void* CPLHTTPSetOptions(void *pcurl, const char * const* papszOptions)
     }
 #endif
 
-    // Enable following redirections.  Requires libcurl 7.10.1 at least.
+    // Enable following redirections.  Requires libcurl 7.19.1 at least.
+#if LIBCURL_VERSION_NUM >= 0x071301
```

Comment:
This version doesn't match the comment.

https://github.com/curl/curl/blob/master/include/curl/curlver.h#L37

Says that it's comparing against 7.13.1.